### PR TITLE
Support extrinsic

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Extrinsic and GA support..
Changed balance is not reflected yet.

### CODE REVIEW

File: packages/react-api/src/Api.tsx [hunk 1]
```diff
@@ -18,10 +18,9 @@ import { formatBalance, isTestChain } from '@polkadot/util';
 import { setSS58Format } from '@polkadot/util-crypto';
 import addressDefaults from '@polkadot/util-crypto/address/defaults';
 
-import typesChain from './overrides/chain';
```
Comment (benxgao) [inline packages/react-api/src/Api.tsx]: No need to remove these two lines, so that we could ensure minimum unnecessary changes/conflicts from polkadot/apps

Comment (KarishmaBothara) [reply packages/react-api/src/Api.tsx]: have put it back

Comment (benxgao) [reply packages/react-api/src/Api.tsx]: May need to use it in `api = new ApiPromise({ provider, registry, signer, typesChain, typesSpec });` at line 129?

Comment (benxgao) [reply packages/react-api/src/Api.tsx]: Otherwise, each time when we rebase from polkadot, we have to resolve the conflict here, thats my concern.

Comment (KarishmaBothara) [reply packages/react-api/src/Api.tsx]: no.. we need to use types specific to our chain.. so I use the types as 
api = new ApiPromise({ provider, registry, signer, types: { ...CENNZRuntimeTypes as any } });

https://polkadot.js.org/api/start/types.extend.html#node-and-chain-specific-types
Single API object can be used to connect to different types of nodes or chains.

Comment (benxgao) [reply packages/react-api/src/Api.tsx]: Gotcha.
